services/web/src/components/form-fields/Uploads.js

```diff
@@ -256,34 +279,30 @@ export default class Uploads extends React.Component {
                     : 'ui icon message upload-dropzone'
                 }
                 style={{ cursor: 'pointer', outline: 0 }}>
-                {loading ? (
-                  <Icon name="rotate" loading />
-                ) : (
-                  this.renderIconForType()
-                )}
+                {loading && <Loader />}
                 <input {...getInputProps()} />
-                <div className="content">
+                <Paper withBorder mt="md" p="md">
                   {loading ? (
-                    <p>Uploading...</p>
+                    <Text>Uploading...</Text>
                   ) : isDragActive ? (
-                    <p>Drop files here...</p>
+                    <Text>Drop files here...</Text>
```

Comment:
This is another one I really don't get what `Text` is doing or why it's needed here.... `p` seems a lot more appropriate

Reply:
Text applies styling, while P does not.  So with Text it will styled with the correct font color etc control in the mantine theme. 

https://mantine.dev/core/text/